### PR TITLE
Added #515 and popup modify

### PR DESCRIPTION
**Pull request by user-promi**

### CODE REVIEW

File: mvx-modules/src/DynamicForm/index.js [hunk 1]
```diff
@@ -107,6 +108,29 @@ export default class DynamicForm extends React.Component {
...
 	}
 
+	onMultiSelectDeselectChange(e, m) {
+		if (this.state[m.key].length > 0) {
+			this.setState({
+				[m.key]: [],
+			});
+		} else {
+			const complete_option_value = [];
+			m.options.map((o, index) => {
```
Comment (moumitahalder) [inline mvx-modules/src/DynamicForm/index.js]: checking needed

File: mvx-modules/src/DynamicForm/index.js [hunk 2]
```diff
@@ -1161,6 +1185,18 @@ export default class DynamicForm extends React.Component {
...
 				input = (
 					<div className="mvx-settings-from-multi-select">
+						{m.select_deselect ? (
+							<div
+								className="mvx-select-deselect-trigger"
+								onClick={(e) => {
+									this.onMultiSelectDeselectChange(e, m);
+								}}
+							>
+								Select / Deselect All
```
Comment (moumitahalder) [inline mvx-modules/src/DynamicForm/index.js]: string support?

File: mvx-modules/src/components/admin/classes/class-mvx-modules-listing.js [hunk 3]
```diff
@@ -488,45 +488,40 @@ class MVX_Module_Listing extends Component {
...
+													{appLocalizer.module_page_string.module17}
+												</div>
+												<div className="mvx-banner-offer">
+													{appLocalizer.module_page_string.module18}
+												</div>
+												<div className="discount-tag">
+													{appLocalizer.module_page_string.module15}
+													<b>{appLocalizer.module_page_string.module20}</b>
+												</div>
+												<p className="">
```
Comment (moumitahalder) [inline mvx-modules/src/components/admin/classes/class-mvx-modules-listing.js]: why blank classname

**Review by moumitahalder — CHANGES_REQUESTED** [review]

**Comment by user-promi** [comment]
Changes Done

**Pull request merged**